app/src/main/java/io/gnosis/safe/ui/StartActivity.kt

```diff
@@ -1,13 +1,87 @@
 package io.gnosis.safe.ui
 
 import android.os.Bundle
+import android.view.View
+import androidx.navigation.Navigation
+import androidx.navigation.ui.setupWithNavController
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import io.gnosis.data.models.Safe
 import io.gnosis.safe.R
+import io.gnosis.safe.databinding.ToolbarSafeOverviewBinding
 import io.gnosis.safe.ui.base.BaseActivity
+import io.gnosis.safe.ui.safe.SafeOverviewNavigationHandler
+import io.gnosis.safe.ui.safe.selection.SafeSelectionDialog
+import io.gnosis.safe.utils.asMiddleEllipsized
+import pm.gnosis.svalinn.common.utils.visible
+import pm.gnosis.utils.asEthereumAddressString
 
-class StartActivity : BaseActivity() {
+class StartActivity : BaseActivity(), SafeOverviewNavigationHandler {
+
+    private val toolbarBinding by lazy {
+        ToolbarSafeOverviewBinding.bind(findViewById(R.id.toolbar_container))
+    }
+    private val toolbar by lazy { findViewById<View>(R.id.toolbar) }
+    private val navBar by lazy { findViewById<BottomNavigationView>(R.id.nav_bar) }
```

Comment:
I wonder if you can access these 2 from the `toolbarBinding`

Reply:
toolbarBinding is toolbar container for accessing views, that display safe data. Navbar is bottom nav view, and toolbar is a container that has both toolbar container (toolbarBinding) and it's artificial shadow. I don't think it is possible to access outer elements from binding.